index.html

```diff
@@ -3,9 +3,20 @@
   <head>
     <title>EST UI Widget</title>
     <link rel="stylesheet" type="text/css" href="style.css">
-    <script type="text/javascript" src="javascript/est-widget.js"></script>
+
+      <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
I was thinking that [this code](https://github.com/GovWizely/est-ui-widget/pull/2/files#diff-4220180f0c119d766886877485f1806aR4) load the JQuery for use inside our anonymous function, and then by calling `window.jQuery.noConflict(true);` [we free up the JQuery](https://api.jquery.com/jquery.noconflict/) by passing the `true` to the noConflict call, we do this so we don't override the original version used in the website that embeds our widget.

But it seems like the JQuery tag is appended indeed in `<head>` but the browser is still complaining `Uncaught ReferenceError: $ is not defined`. i will look into this in more details.